frappe/erpnext
# fix(India): NIL Rated, Exempted and non gst invoices in GSTR-1 report

Added Nil Rated/Exempted and Non-GST section in GSTR-1 report

![image](https://user-images.githubusercontent.com/42651287/149377858-21e8d47d-99c1-4727-b295-bd420357025a.png)

Steps to check:
1. Create Exempted Item (Check "Is Nil Rated and Exempted" check in item master)
2. Make a regular GST Invoice and select the Exempted item
3. Base on the type of customer (Register or Unregistered) and place of supply (Instate or out state) the taxable value should be shown accordingly in the report as shown above
